### PR TITLE
Better error message for https://github.com/guilhermebodin/Dualization.jl/issues/22

### PR DESCRIPTION
**Pull request by dourouc05**
Solves #22

### CODE REVIEW

File: src/supported.jl [hunk 1]
```diff
@@ -65,11 +65,17 @@ function supported_objective(primal_model::MOI.ModelLike) where T
     if !supported_obj(obj_func_type)
         error("Objective functions of type ", obj_func_type," are not implemented")
     end
-    return 
+
+    obj_func = MOI.get(primal_model, MOI.ObjectiveFunction{obj_func_type}())
+    if length(filter(t -> t.coefficient != zero(Float64), obj_func.terms)) == 0
```
Comment (odow) [inline src/supported.jl]:
```Julia
if all(c -> iszero(c.coefficient), obj_func.terms)
```

Comment (dourouc05) [reply src/supported.jl]: Oh, nice! While we're at it, wouldn't `any` be better? It could short-circuit its execution more often, I think.

**Comment by guilhermebodin** [comment]
It was fixed in #23

**Comment by dourouc05** [comment]
And your fix seems cleaner than mine!

**Pull request closed without merging**